locals.tf

```diff
@@ -109,6 +109,7 @@ locals {
   cdn_frontdoor_custom_domain_dns_names = local.enable_cdn_frontdoor && local.enable_dns_zone ? toset([
     for domain in local.cdn_frontdoor_custom_domains : replace(domain, local.dns_zone_domain_name, "") if endswith(domain, local.dns_zone_domain_name)
   ]) : []
+  cdn_frontdoor_origin               = try(var.cdn_frontdoor_origin_override != "", false) ? var.cdn_frontdoor_origin_override : jsondecode(azapi_resource.default.output).properties.configuration.ingress.fqdn
```

Comment:
We don't need the `try` here
```
cdn_frontdoor_origin = var.cdn_frontdoor_origin_override != "" ? var.cdn_frontdoor_origin_override : jsondecode(azapi_resource.default.output).properties.configuration.ingress.fqdn
```

Also perhaps rename the local to `cdn_frontdoor_origin_fqdn`